src/DeviceOperator.ts

```diff
@@ -7,43 +7,15 @@
 import * as vscode from 'vscode';
 
 import {TelemetryContext} from './telemetry';
-import {ProjectHostType} from './Models/Interfaces/ProjectHostType';
-import {askAndNewProject, handleIoTWorkspaceProjectFolder} from './utils';
-
-const impor = require('impor')(__dirname);
-const ioTWorkspaceProjectModule = impor('./Models/IoTWorkspaceProject') as
-    typeof import('./Models/IoTWorkspaceProject');
-const ioTContainerizedProjectModule =
-    impor('./Models/IoTContainerizedProject') as
-    typeof import('./Models/IoTContainerizedProject');
+import {createAndLoadIoTProject} from './utils';

```

Comment:
See my comments in AzureOperator.ts

Reply:
No need to import all. Only use some function in `utils.ts`.